src/WithFileUploads.php

```diff
@@ -96,9 +96,11 @@ protected function cleanupOldUploads()
         $storage = FileUploadConfiguration::storage();
 
         foreach ($storage->allFiles(FileUploadConfiguration::path()) as $filePathname) {
-            $yesterdaysStamp = now()->subDay()->timestamp;
-            if ($yesterdaysStamp > $storage->lastModified($filePathname)) {
-                $storage->delete($filePathname);
+            if ($storage->exists($filePathname)) {
```

Comment:
you could reduce cyclomatic complexity by continue early
```php
if (! $storage->exists($filePathname)) {
    continue;
}

$yesterdaysStamp = now()->subDay()->timestamp;
if ($yesterdaysStamp > $storage->lastModified($filePathname)) {
    $storage->delete($filePathname);
}

Reply:
You're right, this is a better solution. How do I pull in your suggestion in a way that is properly attributed to you? I'm not that familiar with this feature.

Reply:
Just commit it I have enough contributions.

And "my way" is just formating your code different 👍